packages/base/src/UI5Element.js

```diff
@@ -83,22 +84,27 @@ class UI5Element extends HTMLElement {
 	/**
 	 * @private
 	 */
-	_initializeShadowRoot() {
-		if (!this.constructor._needsShadowDOM()) {
-			return;
-		}
+	_initializeContainers() {
+		// Init Shadow Root
+		if (this.constructor._needsShadowDOM()) {
+			this.attachShadow({ mode: "open" });
 
-		this.attachShadow({ mode: "open" });
+			// IE11, Edge
+			if (window.ShadyDOM) {
+				createHeadStyle(this.constructor);
+			}
 
-		// IE11, Edge
-		if (window.ShadyDOM) {
-			createHeadStyle(this.constructor);
+			// Chrome
+			if (document.adoptedStyleSheets) {
+				const style = getConstructableStyle(this.constructor);
+				this.shadowRoot.adoptedStyleSheets = [style];
+			}
 		}
 
-		// Chrome
-		if (document.adoptedStyleSheets) {
-			const style = getConstructableStyle(this.constructor);
-			this.shadowRoot.adoptedStyleSheets = [style];
+		// Init Static area only if needed
+		if (this.constructor._needsStaticArea() && !this.staticAreaInstance) {
+			this.staticArea = new StaticArea(this.constructor.staticAreaTemplate, this.constructor.staticAreaStyles, this.constructor.render);
+			this.staticAreaInstance = this.staticArea.constructor._getInstance();
```

Comment:
the instance of the whole static area, if needed, should be common for all elements, so this.constructor. staticArea